guides/request_lifecycle.md

```diff
@@ -117,7 +117,11 @@ defmodule HelloWeb.HelloHTML do
 end
 ```
 
-Now in order to add templates to this view, we can either define them as function components directly in the module:
+To add templates to this view, we can define them as function components in the module or in separate files. Function components are great for smaller templates and separate files are a good choice when you have a lot of markup or your functions start to feel unmanageable.
+
+Both functional components and template files are compiled to an internal format, there is no runtime or performance difference between them.
```

Comment:
Pretty sure this is true and should ease some new users questions ("are files slower? does it read each time?")

Reply:
I think instead of this line we could say:

> Template files are compiled into the module, so there is no runtime or performance difference between using a separate file or the `~H` sigil.

Reply:
I think using the term "~H sigil" might be premature as we've not discussed `~H` or sigils yet (it is in the next post-code para though). The reader might go "what is a function component? wait what is ~H?" vs reusing the same terms we just outlaid.

How do you feel about this?

> To add templates to this view, we can define them as function components in the module or in separate files. Function components are great for smaller templates and separate files are a good choice when you have a lot of markup or your functions start to feel unmanageable.
>
> Template files are compiled into the module as function components themselves, there is no runtime or performance difference between the two styles.

The only real key point I think we need to make is that the reader shouldn't stress about picking between the two!

Reply:
Perfect!